### DISCUSSION TITLE
Copylot for students is not active

### DISCUSSION THREAD

**Discussion by sillamilla** [community/community#59842] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

Recently, I activated the copilot for a student and did not receive anything or information. The first time I received a refusal because of the data that I did not have in the profile, but the second time everything was successful. How much longer do you have to wait to get a copilot?

![image](https://github.com/community/community/assets/125512524/b66cd3ac-9547-4c5b-b8fa-322cdf9d5a95)
![image](https://github.com/community/community/assets/125512524/7c363fa7-e875-4174-bc26-de15de8e401f)

**Comment by davecheney** [comment, marked as answer]
@sillamilla you will have received a coupon code which entitles you to two years of GitHub Pro, and Copilot. Once that coupon is applied to your account -- you have to do this, it is not automatic -- visiting the Copilot signup screen will show the _free education_ option.

**Reply by digitalAijaz** [reply]
I also have the same issue. I am also getting the message that "Your request for was approved, and your benefits will be available soon". 

Where do I need to search for the coupon code that entitles one to two years of GitHub Pro, and Copilot? Do I get a mail for it?

**Reply by Pascal-H** [reply]
For me it's the same as for @digitalAijaz: I was verified successfully after a failed verification, it now says "Congrats! Your request for @Pascal-H was approved and your benefits will be available soon." (https://education.github.com/discount_requests/application), but I am not on GitHub Pro when I check https://github.com/settings/billing/plans and both my private and institutional emails did not receive a coupon. On the verification page, I also don't see any coupon code. My email settings are set to  "Only receive account related emails, and those I subscribe to."

**Reply by mt221030** [reply]
I have the same problem - didn't get a coupon code or anything accept the confirmation that I was accepted.

**Reply by digitalAijaz** [reply]
@Pascal-H, @mt221030 

I have obtained a GitHub Pro subscription for two years. 
I sent a complaint to GitHub Support (mailto:support@github.com) to address the issue. Their response stated that it may take up to 72 hours for GitHub to grant the associated benefits once the application is approved. I received the benefits approximately 60 hours later.

I hope it will come to you too.

Best Wishes.

**Reply by Pascal-H** [reply]
Thanks a lot for your reply, @digitalAijaz :+1:  
My education status was approved on July 5th and on July 8th I received an email from GitHub Education that I am now having access to the respective benefits. Afterwards, I had to activate Copilot by going to my GitHub profile &rarr; settings &rarr; Billing & Plans &rarr; Plans & Usage &rarr; Add-ons &rarr; GitHub Copilot and click "activate".  
Hope that helps anyone else who is impatiently waiting for the benefits to become unlocked :+1:
